### DISCUSSION TITLE
What are the features of React?

### DISCUSSION THREAD

**Discussion by 0xprincedev** [community/community#20751] [Other Feature Feedback, Questions, & Ideas] [answered]
Let's discuss the feature of react?

JSX
Components
Virtual DOM
One way data binding
High performance

**Comment by devtayyab** [comment, marked as answer]
JSX (JavaScript Syntax Extension)
Virtual DOM
One-way data binding
Performance
Extensions
Conditional statements
Components
Simplicity
